src/utils/fide_ratings.py

```diff
@@ -0,0 +1,167 @@
+    (114, 121, 0.66),
+    (122, 129, 0.67),
+    (130, 137, 0.68),
+    (138, 145, 0.69),
+    (146, 153, 0.70),
+    (154, 162, 0.71),
+    (163, 170, 0.72),
+    (171, 179, 0.73),
+    (180, 188, 0.74),
+    (189, 197, 0.75),
+    (198, 206, 0.76),
+    (207, 215, 0.77),
+    (216, 225, 0.78),
+    (226, 235, 0.79),
+    (236, 245, 0.80),
+    (246, 256, 0.81),
+    (257, 267, 0.82),
+    (268, 278, 0.83),
+    (279, 290, 0.84),
+    (291, 302, 0.85),
+    (303, 315, 0.86),
+    (316, 328, 0.87),
+    (329, 344, 0.88),
+    (345, 357, 0.89),
+    (358, 374, 0.90),
+    (375, 391, 0.91),
+    (392, 411, 0.92),
+    (412, 432, 0.93),
+    (433, 456, 0.94),
+    (457, 484, 0.95),
+    (485, 517, 0.96),
+    (518, 559, 0.97),
+    (560, 619, 0.98),
+    (620, 735, 0.99),
+    (736, 1000, 1.00),
+]
+
+
+def find_win_probability(diff: int) -> float:
+    probability = next(
+        (p for start, end, p in difference_to_probability if start <= abs(diff) <= end),
+        difference_to_probability[-1][2],
+    )
+
+    return probability if diff < 0 else 1 - probability
+
+
+percentage_score_rating_difference: list[int] = [
+    -800,  # 0.00
+    -677,  # 0.01
+    -589,  # 0.02
+    -538,  # 0.03
+    -501,  # 0.04
+    -470,  # 0.05
+    -444,  # 0.06
+    -422,  # 0.07
+    -401,  # 0.08
+    -383,  # 0.09
+    -366,  # 0.10
+    -351,  # 0.11
+    -336,  # 0.12
+    -322,  # 0.13
+    -309,  # 0.14
+    -296,  # 0.15
+    -284,  # 0.16
+    -273,  # 0.17
+    -262,  # 0.18
+    -251,  # 0.19
+    -240,  # 0.20
+    -230,  # 0.21
+    -220,  # 0.22
+    -211,  # 0.23
+    -202,  # 0.24
+    -193,  # 0.25
+    -184,  # 0.26
+    -175,  # 0.27
+    -166,  # 0.28
+    -158,  # 0.29
+    -149,  # 0.30
+    -141,  # 0.31
+    -133,  # 0.32
+    -125,  # 0.33
+    -117,  # 0.34
+    -110,  # 0.35
+    -102,  # 0.36
+    -95,  # 0.37
+    -87,  # 0.38
+    -80,  # 0.39
+    -72,  # 0.40
+    -65,  # 0.41
+    -57,  # 0.42
+    -50,  # 0.43
+    -43,  # 0.44
+    -36,  # 0.45
+    -29,  # 0.46
+    -21,  # 0.47
+    -14,  # 0.48
+    -7,  # 0.49
+    0,  # 0.50
+    7,  # 0.51
+    14,  # 0.52
+    21,  # 0.53
+    29,  # 0.54
+    36,  # 0.55
+    43,  # 0.56
+    50,  # 0.57
+    57,  # 0.58
+    65,  # 0.59
+    72,  # 0.60
+    80,  # 0.61
+    87,  # 0.62
+    95,  # 0.63
+    102,  # 0.64
+    110,  # 0.65
+    117,  # 0.66
+    125,  # 0.67
+    133,  # 0.68
+    141,  # 0.69
+    149,  # 0.70
+    158,  # 0.71
+    166,  # 0.72
+    175,  # 0.73
+    184,  # 0.74
+    193,  # 0.75
+    202,  # 0.76
+    211,  # 0.77
+    220,  # 0.78
+    230,  # 0.79
+    240,  # 0.80
+    251,  # 0.81
+    262,  # 0.82
+    273,  # 0.83
+    284,  # 0.84
+    296,  # 0.85
+    309,  # 0.86
+    322,  # 0.87
+    336,  # 0.88
+    351,  # 0.89
+    366,  # 0.90
+    383,  # 0.91
+    401,  # 0.92
+    422,  # 0.93
+    444,  # 0.94
+    470,  # 0.95
+    501,  # 0.96
+    538,  # 0.97
+    589,  # 0.98
+    677,  # 0.99
+    800,  # 1.00
+]
```

Comment:
Why define it again when we have it in `StaticUtils` (or whatever its name is right now)?